src/Conf.ml

```diff
@@ -632,6 +639,8 @@ let parse_line config ~from s =
            Please use `%s = %s`\n"
           filename lnum name value ;
         update ~config ~from ~name ~value
+    (* special case for disable/enable *)
+    | ["enable"], _ -> update ~config ~from ~name:"disable" ~value:"false"
```

Comment:
This is currently not documented. This PR will necessarily conflict with #302, but maybe add a note to remind us to document the `enable` attribute.